compose/services/docker-compose.database.yml

```
@@ -17,4 +17,11 @@
       MARIADB_PASSWORD : ${DB_PASSWORD}
     networks:
      - internal
-     - web
+     - web
+    
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)